src/test/java/core/basesyntax/tests/OperationTest.java

```diff
@@ -0,0 +1,122 @@
+package core.basesyntax.tests;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.dao.FruitsDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operation.BalanceHandler;
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.operation.PurchaseHandler;
+import core.basesyntax.operation.ReturnHandler;
+import core.basesyntax.operation.SupplyHandler;
+import core.basesyntax.storage.Storage;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationTest {
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/193833057-c060b40c-4b79-475e-a6a6-7380eb261c64.png)
add a separate class with tests for each service, handler etc